src/runtime/hexagon_remote/host_malloc.cpp

```diff
@@ -1,17 +1,41 @@
 #include <android/log.h>
+#include <dlfcn.h>
 #include <fcntl.h>
 #include <pthread.h>
 #include <stdlib.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <unistd.h>
 
+bool use_libdmabuf = false;
+bool use_newer_ioctl = false;
+bool use_libion = false;
+
 namespace {
 
+// DMA-BUF support
+void *dmabufAllocator = NULL;
+
+extern "C" {
```

Comment:
These shouldn't be extern "C", only in an anonymous namspace (which they already are).

Reply:
Done